recipes/xz_utils/all/conanfile.py

```diff
@@ -131,3 +131,4 @@ def package_info(self):
         if not self.options.shared:
             self.cpp_info.defines.append('LZMA_API_STATIC')
         self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.names['pkg_config'] = 'lzma'
```

Comment:
is this really the right name for this recipe? Could you provide more info about this

Reply:
hey @danimtb I checked it here: https://github.com/conan-io/conan-center-index/pull/446#issuecomment-567901865